app/controllers/application_controller.rb

```diff
@@ -13,4 +13,15 @@ def authenticate_superuser
     redirect_to root_path, alert: 'You are not authorized to view this page' unless current_user.superadmin?
   end
 
+
+  def locale_from_header
+    case request.env['HTTP_ACCEPT_LANGUAGE'].scan(/^[a-z]{2}/).first
+      when 'sv'
```

Comment:
[Formatting of case blocks](https://github.com/bbatsov/ruby-style-guide#indent-when-to-case) 😉